flixel/addons/editors/tiled/TiledGroupLayer.hx

```diff
@@ -0,0 +1,57 @@
+package flixel.addons.editors.tiled;
+
+import flixel.addons.editors.tiled.TiledLayer.TiledLayerType;
+import flixel.util.FlxColor;
+
+#if haxe4
+import haxe.xml.Access;
+#else
+import haxe.xml.Fast as Access;
+#end
+
+/**
+ * Copyright (c) 2013 by Samuel Batista
+ * (original by Matt Tuttle based on Thomas Jahn's. Haxe port by Adrien Fischer)
+ * This content is released under the MIT License.
+ */
+class TiledGroupLayer extends TiledLayer
+{
+	public var layers:Array<TiledLayer> = [];
+	var layerMap:Map<String, TiledLayer> = new Map<String, TiledLayer>();
+	
+	public function new(source:Access, parent:TiledMap, noLoadHash:Map<String, Bool>)
+	{
+		super(source, parent);
+		type = TiledLayerType.GROUP;
+		loadLayers(source, parent, noLoadHash);
+	}
+
+	function loadLayers(source:Access, parent:TiledMap, noLoadHash:Map<String, Bool>):Void
```

Comment:
Is this a 1:1 copy of the code in `TiledMap`? I don't like the redundancy this creates...